### PR TITLE
Fix: Backport #5275 to release-1.6

### LINKED ISSUE #5556: [Bug]  collect logs error
**Describe the bug**

fail to collect logs
 
lALPJxuMVGsB3_nNAszNBaI_1442_716

**To Reproduce**

**Expected behavior**

**Screenshots**

**KubeVela Version**
1.6.5

**Cluster information**

1.21

**Additional context**

### PR DESCRIPTION
**Pull request by suwliang3**
### Description of your changes

Update pkg dependency, refer to https://github.com/kubevela/pkg/pull/28  https://github.com/kubevela/kubevela/pull/5275

Fixes #5556 

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Comment by suwliang3** [comment]
@wonderflow @chivalryq  There will be some conflicts in unit tests. Is it okay to resolve these conflicts?Will it affect its original function？

**Comment by chivalryq** [comment]
@suwliang3 Good job. You can try to solve the conflicts. I guess most of them won't affect the function. They are just API incompatibility caused by version bumping.

**Comment by chivalryq** [comment]
Check-diff job failed because the controller-gen version in CI is bumped. Please run `make reviewable` to include the differences. @suwliang3

**Review by wonderflow — CHANGES_REQUESTED** [review]
why there're so many changes while #5275 only changes 6 lines.

**Comment by suwliang3** [comment]
> why there're so many changes while #5275 only changes 6 lines.

It should be the reason for the package version dependency. @chivalryq

File: go.mod [hunk 1]
```diff
@@ -76,10 +76,10 @@ require (
 	github.com/rivo/tview v0.0.0-20220709181631-73bf2902b59a
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/rogpeppe/go-internal v1.9.0
-	github.com/sirupsen/logrus v1.8.1
-	github.com/spf13/cobra v1.4.0
```
Comment (chivalryq) [inline go.mod:80]: Did you add `-u` when running `go get`  @suwliang3 . That would bump the all go modules versions although the backport doesn't need them.

Comment (suwliang3) [reply go.mod:80]: I didn't run `go get`. I run `go mod tidy`.

Comment (chivalryq) [reply go.mod:80]: Why `go mod tidy`? I think `go get github.com/kubevela/pkg@263dc191bf51` is enough. Can you include the essential changes only?

Comment (chivalryq) [reply go.mod:80]: Sorry @suwliang3 I just checked the build process and I found it surely have issues with just `go get github.com/kubevela/pkg@263dc191bf51`. I have just restart the apiserver test which fails.

File: test/e2e-apiserver-test/oam_application_test.go [hunk 2]
```diff
@@ -108,7 +108,7 @@ var _ = Describe("Test oam application rest api", func() {
 			Policies:   app.Spec.Policies,
 			Workflow:   app.Spec.Workflow,
 		}
-		res := post(fmt.Sprintf("/v1/namespaces/%s/applications/%s?dryRun=All", namespace, appName), req)
```
Comment (chivalryq) [inline test/e2e-apiserver-test/oam_application_test.go:111]: Why should we change this?

**Review by chivalryq — APPROVED** [review]
LGTM. cc @wonderflow I checked the changes and most of them is introduced by bumping version of tools like `controller-gen` and dependency in pkg repo. It's OK to merge this.

File: test/e2e-apiserver-test/oam_application_test.go (hunk 2 above)
Comment (suwliang3) [reply test/e2e-apiserver-test/oam_application_test.go:111]: An error is returned in a unit test. Refer to the latest code for modifications.

Comment (chivalryq) [reply test/e2e-apiserver-test/oam_application_test.go:111]: Refer to https://github.com/kubevela/kubevela/pull/5252/files#diff-0144ec5ef7981247aa9d9934f55282f0779e8f08b16b3636602120cdc4307e9a.

**Comment by chivalryq** [comment]
ping @wonderflow

**Review by wonderflow — APPROVED** [review]

**Pull request merged**